stackslib/src/chainstate/stacks/index/marf.rs

```diff
@@ -1035,22 +1035,17 @@ impl<T: MarfTrieId> MARF<T> {
         block_hash: &T,
         path: &TrieHash,
     ) -> Result<Option<TrieLeaf>, Error> {
-        trace!("MARF::get_path({:?}) {:?}", block_hash, path);
+        trace!("MARF::get_path({block_hash:?}) {path:?}");
 
         // a NotFoundError _here_ means that a block didn't exist
         storage.open_block(block_hash).map_err(|e| {
```

Comment:
Change to

```suggestion
        storage.open_block(block_hash).inspect_err(|e| {
```

and don't return a value